### DISCUSSION TITLE
A program to track your progress

### DISCUSSION THREAD

**Discussion by GauravRancike** [scaleracademy/open-source-projects#85] [Ideas]
The idea of project is to create a software that tracks your progress and motivates you to become better everyday.
Here, student can enter multiple inputs such as :
- DAILY SCORE
- TIME TAKEN TO SOLVE
- NO. OF QUESTIONS DONE
- IF NOT QUESTIONS ANY THING ELSE PRODUCTIVE DONE

 What will be the output?
- We design the software such that day the user fails to do anything, the score is deducted.
- Using machine learning algo, the software tells the time level of problem should have taken vs time you spent
- Tells maximum score obtained in week

FUTURE PERSPECTIVE:
Can also tell what level of coder are you.
You can set your goals acc. to number of que or score and it will tell how far you are from score and at what pace you're approaching.

**Comment by tkohli** [comment]
@GauravRancike Great idea, would love to be a part of your project.
We can use something like wakatime and modify it for our usage, looking forward towards this project.
link for wakatime - https://wakatime.com/

**Comment by cy2000yadav** [comment]
For sure a good idea!!

**Comment by kaivalya98** [comment]
That's a great idea!

**Comment by saraswati258** [comment]
Perfect one!!

**Comment by protopuppy** [comment]
We can add Pomodoro technique for time management and increase productivity.

**Comment by DhruvPatel033** [comment]
**That's Amazing idea !**

> best for enhance work productivity and consistency !

**Comment by Rituraj-13** [comment]
That's a great idea 💡& I would love to contribute in it.

**Comment by BhumikaJena** [comment]
Nice idea!
